reference/api/settings.md

```diff
@@ -30,17 +30,18 @@ Get the settings of an index. The index [`uid`](/learn/core_concepts/indexes.md#
 
 ### Response body
 
-| Variable                  | Type      | Description                                                                      | Default value                                                                                     |
-| ------------------------  | --------- | -------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------- |
-| **displayedAttributes**   | [Strings] | Fields displayed in the returned documents                                       | `["*"]` (all attributes)                                                                                                                                |
-| **distinctAttribute**     | String    | Search returns documents with distinct (different) values of the given field     | `null`                                                                                            |
-| **filterableAttributes** | [Strings] | Attributes to use as [filters and facets](/learn/advanced/filtering_and_faceted_search.md)         | `[]`                                                                                              |
-| **rankingRules**          | [Strings] | List of ranking rules sorted by order of importance                              | [A list of ordered built-in ranking rules](/learn/core_concepts/relevancy.md#built-in-rules) |
-| **searchableAttributes**  | [Strings] | Fields in which to search for matching query words sorted by order of importance | `["*"]` (all attributes)                                                                                          |
-| **sortableAttributes**    | [Strings] | Attributes to use when [sorting](/learn/advanced/sorting.md) search results  | `[]`                                                                         |
-| **stopWords**             | [Strings] | List of words ignored by Meilisearch when present in search queries              | `[]`                                                                                              |
-| **synonyms**              | Object    | List of associated words treated similarly                                       | `{}`                                                                                              |
-| **typoTolerance**         | Object    | Typo tolerance settings                                                          | `{}`                                                                                              |
+| Variable                 | Type      | Description                                                                                | Default value                                                                                |
```

Comment:
I can't comment on line 9, but you should add:
`- [Faceting](/reference/api/faceting.md)`